app/src/main/kotlin/edu/stanford/bdh/engagehf/health/components/HealthChart.kt

```diff
@@ -160,8 +160,8 @@ private fun rememberValueFormater(
         CartesianValueFormatter { value, _, _ ->
             val date = when (uiState.selectedTimeRange) {
```

Comment:
Consider this comment from this other PR: https://github.com/StanfordSpezi/SpeziKt/pull/83#discussion_r1720968532